src/moveit_studio_ur_pstop_manager/package.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>moveit_studio_ur_pstop_manager</name>
+  <version>4.0.0</version>
+  <description>Provides a node to monitor the protective stop state of the UR5, and reset protective
+    stops when necessary.</description>
+  <maintainer email="cory.crean@picknik.ai">Cory Crean</maintainer>
+  <license>Proprietary</license>
```

Comment:
Same here, need to change the maintainer to the generic "MoveIt Pro maintainer" and license to BSD 3-Clause. Basically whatever is in the other packages.